istioctl/pkg/dashboard/dashboard.go

```diff
@@ -427,6 +432,10 @@ func istioDebugDashCmd(ctx cli.Context) *cobra.Command {
 			}
 
 			var podName, ns string
+			if revision != "" {
+				labelSelector = "istio.io/rev=" + revision + ", app=istiod"
```

Comment:
More of an open question: I understand that controlz is just an istiod thing and that the helm charts always set `app: istiod`, but why only add `app=istiod` if revision is not empty?

Reply:
also, wouldn't this overwrite existing labelSelectors?

Reply:
Thank you for your attention. I made some changes according to your suggestions, but accidentally compressed and submitted them. The main changes are:
1. Only one of the three parameters --selector, --revision, and name can be specified at the same time
2. When specifying --selector, the label "app=istiod" is also added by default

Reply:
wait a sec. Wouldn't it be better to `opts.AttachControlPlaneFlags(cmd)` and that would automatically add the revision flag. Also, it would use the existing "use this control plane revision" logic on dashboard.go:429

Reply:
Updated, thanks